### DISCUSSION TITLE
Create album from both tags and folder views

### DISCUSSION THREAD

**Discussion by ArturRuta** [immich-app/immich#12901] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The tag and folder views are quite a good way of navigating large photo catalogs, specially when they come from libraries.
They also allow for selectin items but there's no way to create ans album from the selected items. 
It would be a great way to create albums, 

Some sample cases
- Select all photos tagged birthday and from there create an album for the family
- Select all photos tagged birds and from there create an album for friends sharing the same interests
- Select all photos tagged hicking and share them with colleagues doing the same sport
- etc.

The samples could be refined, not necessarily all photos with a given tag sholud be included, the selecctoion buttons in the list could allow for choosing witch ones to include.

Last point... this woud be most useful in the web interface. For the mobile it would be a nice to have...but not really a necessity.

Cooments welcome

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe of #1673 or #9159

**Comment by ArturRuta** [comment]
Excuse me but this is not a duplicate.

#9159 asks for a bulk selection method. This one is not asking for it.
- Tag view already allows individual selection or selection by date like normal view by date.
- Folder view does not allow any kind of selection (that would be needed but this issue is not asking for it.

#1673 ask for a "saved filter" so that when it's invoked the photos matching the criteria are shown. Even photos that have veen added recently as far as they match.

This case is neither of the above. The concept is:
Once positioned on a given tag or a a given folder the phothos shown should be selectable (all of them or just the ones we choose) and it has to be possible to create an album from the selecotion in the very same way selection works for the normal calendar view. 
That album vould be shareable and its contents static (just the photos we selected when creating the album). Again in the very same way normal album creation works in the chronology view.

Hope to have clarified the idea and why it's not the same as the cases mentioned.

Thanks a lot for your interest

**Reply by bo0tzz** [reply]
> Once positioned on a given tag or a a given folder the phothos shown should be selectable (all of them or just the ones we choose) and it has to be possible to create an album from the selecotion in the very same way selection works for the normal calendar view.

That would be covered by #9159

**Comment by ArturRuta** [comment]
Well, not really.
I'm not talking about how to select. I'm talking about what can be done with the selection.

As of now, both the tag view nor the folder wiew are missing the options for
- convert the selectiont into an album whether it's shared or not
- add the selection to a preexisting album
- add the selection to favorites

These options are just missing.
- The only actions alloweb in the tag view are: Create Tag, Edit Tag, Delete Tag
- And as far as the folder view is concernet, well it gives no option at all from the list view, neither selectoin nor action. However if a single photo is open then actions can be done at the individual photo level (add to shared album for example)

So, definitely it seems like something is missing in the tag and folder views, which on the other side is no at all that strange since these features have been added just very recently.

Thanks again for you interest.

**Comment by MD77MD** [comment]
what he means is the ability to creat albums from tags

fir example, in my case if i have photos taged with A,B and C. i would use this feature by creating an album that has tag A and B... or A and C

so the album would look for all pictures that have selected tags... and preferably  add any new photos uploaded to library with those specific tags dynamically.  synology photos have similar future

**Comment by frostworx** [comment]
I'd like to feature request the option as well.

I migrated a huge photo collection (mostly lepidoptera) into immich recently and would love the option to automatically create new albums from existing tags (and add those tagged photos to the album of course).

(some of those tags are also locations. It would also be great if it was possible to add a geolocation to all pictures with tag '$location': selecting all pictures within a tag is already possible, but the option to bulk edit the geolocation is not available)_

**Reply by MD77MD** [reply]
the bulk edit is a great idea... not just for geolocation but for all edits too where possible

**Comment by CorsairVelo** [comment]
I'd like this options as well. Withing a tag view, I need to select a subset of images to add to either a new album or an existing one.  This would be great.
